domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -287,30 +288,32 @@ class TopicListController @Inject constructor(
               storyId,
               story,
               latestStartedChapterProgress,
-              startedChapterProgressList,
+              completedChapterProgressList,
               topic,
               isTopicConsideredCompleted
             ).let {
-              Pair(it.first, it.second)
-              if (completionTimeFilter(it.second))
-                it.first?.let { it1 -> playedPromotedStoryList.add(it1) }
+              numberOfDaysPassed = latestStartedChapterProgress.getNumberOfDaysPassed()
+              if (completionTimeFilter(numberOfDaysPassed)) {
+                playedPromotedStoryList.add(it!!)
```

Comment:
Suggest using ``?.let`` above here & below rather than crashing--we shouldn't be including the story if it doesn't qualify.

Reply:
Done.